changes/22544-move-linux-lock-wipe

```diff
@@ -0,0 +1 @@
+- Remove duplicate scripts from repository
```

Comment:
Please use past tense :)

Reply:
I think at this point I just need a little picture of you on my desk with a speech bubble that says "Please use past tense :)"

Reply:
Fixed 😀